config/rbac/role.yaml-e

```diff
@@ -0,0 +1,109 @@
+
```

Comment:
I think this file must be removed and was added by accident?

Reply:
This file should not be tracked. I somehow added it.